### PR TITLE
Updating the README with more descriptive detail

### PR DESCRIPTION
**Pull request by lumberyard-employee-dm**
Added information about the responsibilities of the SIG-Core group.
Added links to the list of Open issues for the SIG-Core group, a link to the charter, as well information about the Chair and Co-Chair.

Furthermore information about how to contact the SIG-Core group, as well as the time of triage sessions take place have been added.

Signed-off-by: lumberyard-employee-dm <56135373+lumberyard-employee-dm@users.noreply.github.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,46 @@
...
+
 
 # General Resources
+
+# Open 3D Engine Special Interest Group for Core Systems (SIG Core)
+
+This repository holds materials for the Open 3D Engine Foundation Special Interest Group for Core Systems (SIG-Core).
+
+## Description of SIG Core
+We manage the Core Systems needed by O3DE libraries to fulfill the engines requirements. Primarily low-level systems such as memory management, application startup and file input/output are implemented by. See the [SIG's charter](governance/SIG%20Core%20Charter.md) for full details.
```
Comment (lmbr-pip) [inline README.md:14]:
```suggestion
We manage the Core Systems needed by O3DE and its libraries to fulfill the engines requirements. The SIG primarily owns low-level systems such as memory management, application startup and file input/output. See the [SIG's charter](governance/SIG%20Core%20Charter.md) for full details.
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,46 @@
...
+
+## What the SIG does:
+
+* Contributes to changes to O3DE Core Systems in library(AzCore/AzFramework/AzToolsFramework/AzGameFramework).
```
Comment (lmbr-pip) [inline README.md:18]: Doesn't the SIG owns most of AzCore/AzFramework? It seems odd just to call out AzGameFramework IMHO

**Review by lmbr-pip — COMMENTED** [review]
LG overall.

Thanks for making these updates.

File: README.md (hunk 2 above)
Comment (lumberyard-employee-dm) [reply README.md:18]: Yeah it is quite specific.
I'll remove it from the list of Core Systems

**Pull request merged**